Sources/Player/OfflineEngine/Internal/Storage/DBStorage/DBOfflineEntryDTO.swift

```diff
@@ -0,0 +1,104 @@
+import Foundation
+import GRDB
+
+// MARK: - DBOfflineEntryDTO
+
+struct DBOfflineEntryDTO: Codable, FetchableRecord, PersistableRecord {
```

Comment:
[suggestion] if we agree on a different naming from last PR, then let's apply it also for the entities here.